dissect/target/filesystems/config.py

```diff
@@ -122,7 +123,8 @@ def _convert_entry(self, file_entry: FilesystemEntry, *args, **kwargs) -> Config
         entry = file_entry
         config_parser = None
         try:
-            config_parser = parse(entry, *args, **kwargs)
+            target_path = TargetPath(entry.fs, entry.path)
```

Comment:
reviewer: Perhaps there is a nicer way?

Reply:
Perhaps `entry.fs.path(entry.path)`?

I still think mixing the filesystem layer with the config layer is very confusing to begin with, but this is probably at least closer to the intended API.

Reply:
`entry.fs.path(entry.path)` is indeed a bit nicer.

I still think mixing the filesystem layer with the config layer is very confusing to begin with, but this is probably at least closer to the intended API.

Yeah at least with this change the parsers are decoupled from the config layer.

I did not want to make more substantial changes without fully understanding why it was written this way, and the primary author
was on holiday though. I will discuss with him and possibly create a ticket depending on the outcome.